extensions/standard-processors/tests/unit/ExtractTextTests.cpp

```diff
@@ -180,3 +181,36 @@ TEST_CASE("Test usage of ExtractText in regex mode", "[extracttextRegexTest]") {
 
   LogTestController::getInstance().reset();
 }
+
+#ifndef NO_MORE_REGFREEE
```

Comment:
Is this `#ifndef` required? It should work in either case if I understand correctly.

Reply:
Good catch, it was turned off while testing locally, I readded it in affce89dfb183b10598bf0e8a4351a0e92db3fb2